bucket/git-credential-manager-core.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "2.0.475",
+    "description": "Git Credential Manager Core (GCM Core) is a secure Git credential helper built on .NET that runs on Windows and macOS. Linux support is in an early preview.",
```

Comment:
Does not make sense to have application name

Reply:
ok will fix.